server/config/options.go

```diff
@@ -93,13 +102,18 @@ var DefaultConfig = Config{
 		WriteEnabled: true,
 	},
 	Tracing: TracingConfig{
-		Enabled: true,
+		Enabled: false,
```

Comment:
So we are keeping it disabled by default?

Reply:
Yes as the profiler would log recurring errors on developers laptops. Those laptops would likely have no opentracing or datadog tracing sink installed.